guides/INSTALLING.md

```diff
@@ -1,72 +1,118 @@
+* [Cypher Shell](https://neo4j.com/docs/operations-manual/current/tools/cypher-shell/)
+  - `$ brew install cypher-shell`
 * [Node.js and npm](https://nodejs.org/en/), install the LTS version with:
-
-  - `brew install node@12`
-  
-  - copy `export PATH="/usr/local/opt/node@12/bin:$PATH"` onto the end of your `.zshrc` or `.basrc` file and restart your terminal.
+  - `$ brew install node@12`
+  - append the following to your [startup file](#startup-file)
+    - `export PATH="/usr/local/opt/node@12/bin:$PATH"`
 
 * [dp-compose](https://github.com/ONSdigital/dp-compose)
 
-  `git clone git@github.com:ONSdigital/dp-compose`
+  Clone the repo:
 
-  NB. See `dp-compose` README for configuration of Docker Desktop resources
+  `$ git clone git@github.com:ONSdigital/dp-compose`
+
+  then see the `dp-compose` README file for configuration of Docker Desktop resources
 
   - Services for Web
     - Elasticsearch 2.4.2
     - Highcharts
     - Postgres
-    - MathJax 
+    - MathJax
   - Services for CMD
     - MongoDB
     - Elasticsearch 5 (on non-standard port)
     - Kafka (plus required Zookeeper dependency)
     - Neo4J
-* [go v1.13](https://golang.org/doc/install) (You should already have `go1.13` installed from the Prerequisites section in the `dp-cli` `README.md`)
+
 * [GoConvey](https://github.com/smartystreets/goconvey#installation)
 * [GhostScript](https://www.ghostscript.com/download.html) - Required for [Babbage](https://github.com/onsdigital/babbage)
-
   - `brew install ghostscript`
-* [Vault](https://www.vaultproject.io/intro/getting-started/install.html) - This will be required for running Florence.
-
-  - `brew install hashicorp/tap/vault`
-
-* [jq](https://stedolan.github.io/jq/) - installed with `brew install jq`
-  - `brew install jq`
-  - This isn't essential, but it's a useful tool for debugging website content.
+* [Vault](https://learn.hashicorp.com/tutorials/vault/getting-started-install)
+  - `$ brew install hashicorp/tap/vault`
+* [jq](https://stedolan.github.io/jq/) - a handy JSON tool
+  - `$ brew install jq`
+* [yq](https://github.com/mikefarah/yq) - a handy YAML tool
+  - `$ brew install yq`
 
 Return to the [Getting Started](https://github.com/ONSdigital/dp/blob/master/guides/GETTING_STARTED.md) guide for next steps.
 
---------------
+---
 
 ## Clone and run each service
 
 Clone the GitHub repos for [web](#web), [publishing](#publishing) and/or [CMD](#cmd).
 
-- [Web](#web) will be enough strictly to work on website content types other than filterable datasets (e.g. bulletins, articles, timeseries, datasets). 
+- [Web](#web) will be enough to work on website content other than filterable datasets (e.g. bulletins, articles, timeseries, datasets).
+
+- [Publishing](#publishing) gives you the ability to update, preview and publish content - required for changes to any part of the publishing system.
+
+- [CMD](#cmd) services support the import, publish and filterable dataset journeys.
```

Comment:
I think these descriptions should give a very high level overview of what each "mode" does. Nothing heavy, just enough for context. Something like:
```
Clone the services
web - These apps make up the public facing website providing read only access to published content.

Publishing - Running  in "publishing mode" gives you all the features of web but also provides and internal interface for editing, creating deleting content. All content is encrypted and requires authentication.
```